openreview/venue/invitation.py

```diff
@@ -3095,4 +3095,100 @@ def set_SAC_ethics_flag_invitation(self, sac_ethics_flag_duedate=None):
             if sac_ethics_flag_duedate:
                 invitation.edit['invitation']['expdate'] = tools.datetime_millis(sac_ethics_flag_duedate)
 
-            self.save_invitation(invitation, replacement=True)
\ No newline at end of file
+            self.save_invitation(invitation, replacement=True)
+
+    def set_group_recruitment_invitations(self, committee_name):
+
+        venue_id = self.venue_id
+        venue = self.venue
+
+        invitation = Invitation(id=venue.get_group_recruitment_id(committee_name),
+            invitees=[venue_id],
+            readers=[venue_id],
+            writers=[venue_id],
+            signatures=['~Super_User1'],
+            process=self.get_process_content('process/group_recruitment_process.py'),
+            content={
+                'committee_name': { 'value': committee_name },
+                'official_committee_roles': { 'value': venue.get_committee_names()}
+            },
+            edit={
+                'signatures': [venue_id],
+                'readers': [venue_id],
+                'writers': [venue_id],
+                'content': {
+                    'inviteeDetails': { 
+                        'value': {
+                            'param': {
+                                'type': 'string',
+                                'maxLength': 200000,
+                                'input': 'textarea',
+                                'optional': True,
+                                'markdown': True
+                            }
+                        }
+                    }
+                },
```

Comment:
Why not make the invitation modify group.members? This is what the group editor currently does when you add and remove members from a group.

Reply:
We do not want to add them directly to the /Invited group, in case the email or profile ID is not valid. There are also checks we do in the process function (like if the user was already invited to another role, for example.) The process function checks the user can be invited, sends the email and adds the user to the group

Reply:
@melisabok correct me if I am wrong, but this field should be just a textarea like we have in the venue request recruitment, right? The preprocess/process functions should check that they are valid profile IDs or emails.

Reply:
Right, I don't think we need a profile search widget here, because the PCs will invite the reviewers in bulk so they will provide the list of emails. 

I think we could have another invitation that let them invite a reviewer one by one and use the profile search widget there.

Reply:
OK, I guess that makes sense. 1 invitation for bulk add, 1 invitation for adding individuals and 1 invitation for removing individuals.